portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,102 @@
+body {
+  background-color: #CCC5B9;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
+  background-color: white;
+  margin: 0px auto;
+  padding: 2.5% 5% 5% 5%;
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#name {
+  text-align: center;
+}
+
+#hello-world-greeting {
+  color: #495057;
+  font-family: 'Press Start 2P', cursive;
+  font-size: 12px;
+}
+
+p {
+  text-align:justify
+}
+
+section header h1 {
+  display: inline-block;
+  margin: 0;
 }
+
+.time-span {
+  float: right;
+  font-style: italic;
+}
+
+/* Adding side border to the left of each section from Education, Experience & Projects */
+#education section, 
+#experience section, 
+#projects section {
+  border-left: 2px solid #CCC5B9;
+  margin-left: 1.5em;
+  padding-left: 1em;
+}
+
+/* Navigation Bar */
+.navbar-list {
+  background-color: #495057;
+  list-style-type: none;	/* remove the bullets */
+  margin: 0;
+  padding: 0;
+}
+
+.navbar-list li {
+  border: thin solid white;
+  display: inline;	/* Create a horizontal navbar */
+}
+
+.navbar-list a {
+  color: #F8F9FA;
+  padding: 0px 20px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.navbar-list a:hover {
+  background-color: #212529;
+  color: #F8F9FA;
+}
+
+.active-nav-section {
+  background-color: #ADB5BD;
+  color: #212529;
+}
+
+/* Image Gallery */
+.image-container {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-around;
+}
+
+.image-container img {
+  border-radius: 10%;
+  max-width: 200px;
+}
+
+.image-container img:hover {
+  border: thick outset rgb(114, 119, 124);
+}
```

Comment:
Nit: Adding the outline makes the images jump around which looks a bit odd. Maybe add a transparent border to the image-container when the user is not hovering over an image.

Reply:
Good point! Done!